extensions/markdown-language-features/server/package.json

```diff
@@ -14,11 +14,11 @@
     "out/**/*.js"
   ],
   "dependencies": {
+    "@vscode/l10n": "^0.0.7",
```

Comment:
why did you revert?

Reply:
Looks .like 0.0.8 _is_ there but it's not the latest anymore. Weird. Well in any case, https://github.com/microsoft/vscode-l10n/pull/48 this should get us back on track

Reply:
> Can you also look into including encoding as a dependency?

we use v2 of node-fetch and `encoding` appears to be optional. Do you need it?

Reply:
@TylerLeonhardt The build was failing previously because it was missing but looks like that's fixed now